src/utils/daos_autotest.c

```diff
@@ -32,6 +33,11 @@ clock_t	end;
 
 /** generated container UUID */
 uuid_t		cuuid;
+
+/** generate container UUID for aux cont */
+uuid_t		cuuid2;
+uuid_t		cuuid3;
```

Comment:
We could just use labels now for all the containers.

Reply:
Labels are now utilized for containers.